### PR TITLE
Add type definition for phoneNumberPrefix

### PR DESCRIPTION
**Pull request by koshini**
Add type definition for `phoneNumberPrefix` for `Country` to reflect [changes in the country_service API](https://github.com/Shopify/country-service/pull/42/files)

### CODE REVIEW

File: packages/address/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/address",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (nimzco) [inline packages/address/package.json]: @TheMallen, do we have to keep the version bump in a separate PR / commit?

Comment (TheMallen) [reply packages/address/package.json]: yes

Comment (TheMallen) [reply packages/address/package.json]: The `release` line tool will find all changed packages and let the person who runs it choose new versions

**Comment by koshini** [comment]
Reverted the version bump! 🔙  @nimzco @TheMallen
